pom.xml

```diff
@@ -658,7 +658,7 @@
           <source>${maven.compiler.source}</source>
           <quiet>true</quiet>
           <encoding>utf-8</encoding>
-          <docEncoding>utf-8</docEncoding>
+          <docencoding>utf-8</docencoding>
```

Comment:
This parameter is pointless because it duplicates `project.reporting.outputEncoding`. So is `encoding` which even contrdicts `project.build.sourceEncoding`. Both must go away.

Reply:
@michael-o you mean we shall delete both <encoding>utf-8</encoding> and <docEncoding>utf-8</docEncoding> ?

Reply:
Yes, this is my understand. The same issue is with commons parent. If someone else of the committers proves me something different. It is just dedundancy. @britter @bodewig what is your opinion?

Reply:
@kinow  Here.